src/namespaceActions.js

```diff
@@ -32,7 +32,9 @@ const flattenWhenNode = predicate => function flatten(
 };
 
 const flattenActionMap = flattenWhenNode(isPlainObject);
-const flattenReducerMap = flattenWhenNode(node => isPlainObject(node) && !hasGeneratorInterface(node));
+const flattenReducerMap = flattenWhenNode(
+  node => isPlainObject(node) && !hasGeneratorInterface(node)
```

Comment:
@yangmillstheory the renaming to `flattenWhenNode` broke the tests because line was too long.
Is this notation fine, or do you want to have only the expression in separate line?